test/testdata/secrets-samples/edgeconnect-tenant.yaml

```diff
@@ -0,0 +1,7 @@
+name: name-of-your-edge-connect
+tenantUid: abc1234
```

Comment:
```suggestion
tenantUid: abc12345
```

Reply:
why?

Reply:
Tenant UUIDs always have 5 digits, would make the sample a bit more consistent.

Reply:
fixed.